psi4/driver/driver_cbs.py

```diff
@@ -1469,44 +2102,38 @@ def cbs(func, label, **kwargs):
 
     # Call schemes for each portion of total energy to 'place orders' for calculations needed
```

Comment:
This comment isn't clear. Is "call schemes" a noun? Are we calling schemes, so "call" is a verb? What is "place orders"? I think that this part of the code is trying to compute the necessary computations, but I can't be sure.

Reply:
I'm hesitant to do much about this, as I imagine DDD will use a different way of requesting "jobs".